xsuite/src/cli/newWalletCmd.ts

```diff
@@ -71,11 +74,13 @@ const action = async ({
             });
       data = newWallet.toJSON();
     }
-    keystore = Keystore.createFile_unsafe(walletPath, password, data);
+    keystore = Keystore.createFile_unsafe(walletPath, password, data, shard);
```

Comment:
Reviewing your PR made me realize that there was an issue in the code that I fixed in this: https://github.com/arda-org/xSuite/pull/76

Would it be possible to rebase this branch on the dev branch?

It would then make the rest of this PR simpler.